.travis.yml

```diff
@@ -24,7 +27,12 @@ before_script:
 
 script:
   - make -C mpy-cross
-  - make -C minimal test
+  - make -C minimal CROSS=1 build/firmware.bin
+  - ls -l minimal/build/firmware.bin
+  - tools/check_code_size.sh
+  - mkdir -p ${HOME}/persist
+  # Save new firmware for reference, but only if building a main branch, not a pull request
+  - '[ "$TRAVIS_PULL_REQUEST" = "false" ] && cp minimal/build/firmware.bin ${HOME}/persist/'
```

Comment:
Yup, that condition was added as the last step, after I tested other functionality, and oops, it bit. Should be fixed now.